### PR TITLE
OEL-1863: Changed selector from FilePattern.

### PR DESCRIPTION
**Pull request by tibi2303**
Jira issue(s):
- [D8AGE-???](https://webgate.ec.europa.eu/CITnet/jira/browse/D8AGE-???)

### CODE REVIEW

File: tests/src/PatternAssertion/FilePatternAssert.php [hunk 1]
```diff
@@ -53,7 +54,7 @@ protected function assertBaseElements(string $html, string $variant): void {
    *   The crawler.
    */
   protected function assertFile(array $expected, Crawler $crawler): void {
-    $file = $crawler->filter('body > div.mt-4 > div.border.rounded > div:first-child');
+    $file = $crawler->filter('body > .bcl-file-container > .bcl-file.border.rounded > div:first-child');
```
Comment (AaronGilMartinez) [inline tests/src/PatternAssertion/FilePatternAssert.php]: The change with the wrapper selector is good, although I think the ticket is asking to remove this direct child selector to allow the use of this assert in the whitelabel person test like content banner assert here: https://github.com/openeuropa/oe_whitelabel/blob/cc8f899dd188e24d3e4b4c83a15012339282c4a1/tests/src/Functional/PublicationContentRenderTest.php#L46

Can you check, please?
If it's the case there may be other places in this assert class with direct body child selector.

**Review by AaronGilMartinez — CHANGES_REQUESTED** [review]

File: tests/src/PatternAssertion/FilePatternAssert.php (hunk 1 above)
Comment (tibi2303) [reply tests/src/PatternAssertion/FilePatternAssert.php]: you are right. I removed the direct body > selectors.

**Review by AaronGilMartinez — CHANGES_REQUESTED** [review]
Changes are ok, only needs a rebase

**Review by AaronGilMartinez — DISMISSED** [review]

File: tests/src/PatternAssertion/FilePatternAssert.php [hunk 2]
```diff
@@ -46,8 +46,9 @@ protected function assertBaseElements(string $html, string $variant): void {
     $this->assertElementExists('svg.icon--2xl', $crawler);
     $this->assertElementExists('svg.icon--file', $crawler);
```
Comment (drishu) [inline tests/src/PatternAssertion/FilePatternAssert.php]: I think we should change this, lets just check the presence of the svg, or even remove these lines (See what makes sense) as we have the IconPatternAssert test

**Review by drishu — CHANGES_REQUESTED** [review]

File: tests/src/PatternAssertion/FilePatternAssert.php (hunk 2 above)
Comment (tibi2303) [reply tests/src/PatternAssertion/FilePatternAssert.php]: image

it's always visible inside the file but indeed we can make it 1 line.

**Review by drishu — APPROVED** [review]

**Review by AaronGilMartinez — APPROVED** [review]

**Pull request merged**